week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,21 @@
+
 var mentor1 = "Daniel";
+var mentor1UpperCase = mentor1.toUpperCase();
+console.log(" HELLO " + mentor1UpperCase);
+
 var mentor2 = "Irina";
+var mentor2UpperCase = mentor2.toUpperCase();
+console.log(" HELLO " + mentor2UpperCase);
+
 var mentor3 = "Mimi";
+var mentor3UpperCase = mentor3.toUpperCase();
+console.log(" HELLO " + mentor3UpperCase);
+
 var mentor4 = "Rob";
+var mentor4UpperCase = mentor4.toUpperCase();
+console.log(" HELLO " + mentor4UpperCase);
+
 var mentor5 = "Yohannes";
+var mentor5UpperCase = mentor5.toUpperCase();
+console.log(" HELLO " + mentor5UpperCase);
+
```

Comment:
You see a lot of repeated code here. Can you use functions to achieve the same without as much repetition?